src/components/aria-live/README.md

```diff
@@ -0,0 +1,37 @@
+# MdAriaLive
```

Comment:
Yes, MdLiveAnnouncer would probably be better. Do we want this to be a component or a service (or both)?